package-lock.json

```diff
@@ -3364,7 +3364,7 @@
     },
     "ansi-escapes": {
       "version": "1.4.0",
-      "resolved": "https://registry.npmjs.org/ansi-escapes/-/ansi-escapes-1.4.0.tgz",
+      "resolved": "http://registry.npmjs.org/ansi-escapes/-/ansi-escapes-1.4.0.tgz",
```

Comment:
As you didn't change the `package.json` the `package-lock.json` should be updated. You can use `npm install --no-save` to update your deps without updating the package lock.